assets/css/main.css

```diff
@@ -0,0 +1,64 @@
+
+h1 {
+  font-family: helvetica sans-serif;;
+  border-bottom: 1px dashed #bbb;
+}
+
+#github-banner {
+  display: none;
+}
+
+#header {
+  height: 125px;
+  width: 200px;
+  padding: 10px;
+}
+
+#content {
+  min-height: cal(100vh - 200px);
```

Comment:
Yeah it seems to behave the same when I fix the spelling (and this is a current issue with all of the templates it would seem, so main site, docs, etc...)